### PR TITLE
W-14027159 add pdk spotlit gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc [hunk 1]
```diff
@@ -1,18 +1,24 @@
-= Policy Development Kit Overview
+= Flex Gateway Policy Development Kit (PDk) Overview
```
Comment (chris-campbell9) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc]:
```suggestion
= Flex Gateway Policy Development Kit (PDK) Overview
```

File: pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+Flex Gateway Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
 
-PDK provides step-by-step instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway by using API Manager or YAML configuration files.
+PDK provides instructions to guide you through the policy development lifecycle, from implementing your Rust source code to releasing your policy on Exchange. Like included policies, you can apply custom policies to Flex Gateway running in Connected Mode by using API Manager or to Flex Gateway running in Local Mode by using YAML configuration files.
 
-PDK also provides template policies to help you get started with policy development.
+PDK also provides xref:pdk::policies-pdk-develop-custom-policies.adoc[example policies] to help you get started with policy development.
 
-== Before You Begin
+Watch the PDK Spotlight video to see a quick overview of PDKß.
```
Comment (chris-campbell9) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc]:
```suggestion
Watch the PDK Spotlight video to see a quick overview of PDK.
```

File: pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+= Flex Gateway Policy Development Kit (PDk) Overview
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
 Policies enforce regulations to help manage security, control traffic, and improve API adoption. Flex Gateway provides many xref:gateway::policies-included-directory.adoc[included policies], but they might not cover all your specific business needs.
 
-The Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
+Flex Gateway Policy Development Kit (PDK) is a software development kit that enables Flex Gateway users to develop custom policies with ease.
```
Comment (chris-campbell9) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc]:
```suggestion
Flex Gateway Policy Development Kit (PDK) is a software development kit that enables you to develop custom policies with ease.
```

File: pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc (hunk 2 above)
Comment (chris-campbell9) [reply pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc]: Also, is "Spotlight" supposed to be capitalized?

Comment (glenn-rodgers-sf) [reply pdk/1.0/modules/ROOT/pages/policies-pdk-overview.adoc]: oof, forgot to push

**Review by chris-campbell9 — APPROVED** [review]

**Pull request merged**